tools/plugin_template/pluma-plugin.c

```diff
@@ -25,160 +25,286 @@
 #include "##(PLUGIN_MODULE)-plugin.h"
 
 #include <glib/gi18n-lib.h>
-#include <pluma/pluma-debug.h>
+#include <gmodule.h>
+#include <libpeas/peas-activatable.h>
+##ifdef WITH_CONFIGURE_DIALOG
+#include <libpeas-gtk/peas-gtk-configurable.h>
+##endif
 
-#define WINDOW_DATA_KEY	"##(PLUGIN_ID.camel)PluginWindowData"
+#include <pluma/pluma-window.h>
+#include <pluma/pluma-debug.h>
 
-#define ##(PLUGIN_ID.upper)_PLUGIN_GET_PRIVATE(object)	(G_TYPE_INSTANCE_GET_PRIVATE ((object), TYPE_##(PLUGIN_ID.upper)_PLUGIN, ##(PLUGIN_ID.camel)PluginPrivate))
+static void peas_activatable_iface_init (PeasActivatableInterface *iface);
```

Comment:
We do not use `PeasActivatable`  anymore. Please see https://github.com/mate-desktop/pluma/blob/f628a6ef38b7c42febf8fbc450d46ab1f5461f6f/plugins/trailsave/pluma-trail-save-plugin.c#L31 for example for the current way to write a plugin.